docs/source/conf.py

```diff
@@ -86,6 +87,12 @@
 :github_url: deeppavlov/dialog_flow_framework
 """
 
+# Add pictures on the preview buttons
+html_static_path = ["static"]
+
+nbsphinx_thumbnails = {
+    "examples/*/1_*": "_static/hippo.jpg",
```

Comment:
Attention!! Will not add binary files into the repo

Reply:
Removed